website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/how-it-works.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Retroactive Public Goods Funding - How it works
+description: Overview of how retroactive public goods funding works.
+sidebar_label: How it works
+sidebar_position: 2
+---
+
+Are you interested in running a retroactive funding round in your community? 🧐
+
+You’re on the right page! 🥳 We aim to provide guidance and an overview of the process involved.
+
+Besides a few technical & operational requirements, we are very open to new ideas of running rounds. If you have a new idea, feel free to share it with us!
+
+## What is Retroactive Public Goods Funding?
+
+Retroactive Public Goods Funding (RPGF) empowers communities to allocate resources towards funding Public Goods. For those who don't know much about Retroactive Public Goods Funding, [start here](what-is-retroactive-public-goods-funding.md).
+
+## Timelines
+
+Timelines of running a RPGF round varies widely depending on the structure of the round, as well as your team’s bandwidth to organise, promote, deploy & operate the round.
+
+We typically recommend **8-10 weeks** of lead time to prepare to run a RPGF round.
+
+## Time commitment
+
+Requirements here vary widely depending on the structure of the round & team size.
+
+This includes technical & operational work:
+
+- **Technical work**
+  Eg. Configuring wallets, deploying contracts & deploying the website
+  We’d recommend allocating approximately 200 hours of developer time to prepare for a round.
+- **Operational work**
+  Marketing, communications, curating & reviewing project applications & community support
```

Comment:
<img width="803" alt="image" src="https://github.com/privacy-scaling-explorations/maci/assets/9023842/5245c4cc-5ae4-4476-9437-09e035221ec4">

also add a `-` after the title?